cmd/influxd/run.go

```diff
@@ -89,6 +90,31 @@ func execRun(args []string) {
 			go func() { log.Fatal(http.ListenAndServe(config.ApiHTTPListenAddr(), sh)) }()
 		}
 		log.Printf("DataNode#%d running on %s", s.ID(), config.ApiHTTPListenAddr())
+
+		// Spin up any Graphite servers
+		for _, c := range config.Graphite {
+			if !c.Enabled {
+				continue
+			}
+
+			var g graphite.GraphiteServer
+			if strings.ToLower(g.Protocol) == "tcp" {
+				g = graphite.NewTcpGraphiteServer(s)
+			} else {
+				g = graphite.NewUdpGraphiteServer(s)
+			}
```

Comment:
Can you use `TCP` and `UDP` per the Go style guide?

https://github.com/golang/go/wiki/CodeReviewComments#initialisms